package.json

```diff
@@ -133,7 +133,9 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render dist:js build:api test build:docs bundler postinstall types",
+    "build": "npm-run-all build:sass sass:render",
```

Comment:
Can you explain the point of splitting these commands up? This could cause some confusion during local development, due to how our build scripts differ in all other components

Reply:
See Discussion https://github.com/AlaskaAirlines/WC-Generator/discussions/523, this is also referenced in the commit body https://github.com/AlaskaAirlines/auro-popover/pull/64/commits/4567d230e466e546a452ce8afaac8fcfe280a9ee